snapcraft/plugins/dotnet.py

```diff
@@ -140,3 +151,64 @@ def _get_appname(self):
             if fnmatch.fnmatch(file, '*.??proj'):
                 return os.path.splitext(file)[0]
                 break
+
+    def _get_version_metadata(self, version):
+        jsonData = self._get_dotnet_release_metadata()
+        package_data = list(filter(lambda x: x.get('version-runtime')
+                            == version, jsonData))
+
+        if not package_data or len(package_data) < 1:
```

Comment:
This is redundant: just use `if not package_data`. Python's [truth value testing](https://docs.python.org/3/library/stdtypes.html#truth) takes the length into account.